evm/interchainMultisig.js

```diff
@@ -0,0 +1,563 @@
+    }
+}
+
+const signCallsBatch = async (batchId, calls, wallet) => {
+    const callsBatchData = encodeInterchainCallsBatch(batchId, calls);
+    const hash = arrayify(keccak256(callsBatchData));
+
+    return wallet.signMessage(hash);
+};
+
+let calls = [];
+
+async function processCommand(_, chain, options) {
+    const {
+        contractName,
+        address,
+        action,
+        symbols,
+        tokenIds,
+        limits,
+        mintLimiter,
+        newMultisig,
+        recipient,
+        target,
+        calldata,
+        nativeValue,
+        withdrawAmount,
+        privateKey,
+        yes,
+        offline,
+    } = options;
+
+    const newSigners = options.newSigners.split(',');
+    const newWeights = options.newWeights.split(',').map(Number);
+    const newThreshold = Number(options.newThreshold);
+
+    const contracts = chain.contracts;
+    const contractConfig = contracts[contractName];
+
+    let multisigAddress;
+
+    if (isValidAddress(address)) {
+        multisigAddress = address;
+    } else {
+        if (!contractConfig?.address) {
+            throw new Error(`Contract ${contractName} is not deployed on ${chain.name}`);
+        }
+
+        multisigAddress = contractConfig.address;
+    }
+
+    const rpc = chain.rpc;
+    const provider = getDefaultProvider(rpc);
+
+    const wallet = await getWallet(privateKey, null, options);
+
+    printInfo('Contract name', contractName);
+    printInfo('Contract address', multisigAddress);
+
+    const multisigContract = new Contract(multisigAddress, IInterchainMultisig.abi, wallet);
+
+    printInfo('InterchainMultisig Action', action);
+
+    if (prompt(`Proceed with action ${action} on chain ${chain.name}?`, yes)) {
+        return;
+    }
+
+    switch (action) {
+        case 'signers': {
+            const signerEpoch = await multisigContract.epoch();
+            const signerHash = await multisigContract.signerHashByEpoch(signerEpoch);
+
+            printInfo('Signer epoch', signerEpoch);
+            printInfo('Signer hash', signerHash);
+
+            return;
+        }
+
+        case 'setTokenMintLimits': {
+            const symbolsArray = JSON.parse(symbols);
+            const limitsArray = JSON.parse(limits);
+
+            if (!isNonEmptyStringArray(symbolsArray)) {
+                throw new Error(`Invalid token symbols: ${symbols})}`);
+            }
+
+            if (!isNumberArray(limitsArray)) {
+                throw new Error(`Invalid token limits: ${limits}`);
+            }
+
+            if (symbolsArray.length !== limitsArray.length) {
+                throw new Error('Token symbols and token limits length mismatch');
+            }
+
+            const multisigTarget = chain.contracts.AxelarGateway?.address;
+
+            if (!isValidAddress(multisigTarget)) {
+                throw new Error(`Missing AxelarGateway address in the chain info.`);
+            }
```

Comment:
Also can use this function here instead of separate checks:
https://github.com/axelarnetwork/axelar-contract-deployments/blob/main/evm/utils.js#L403